micropython/drivers/imu/bmi270/manifest.py

```diff
@@ -1,2 +1,2 @@
-metadata(description="BOSCH BMI270 IMU driver.", version="1.0.0")
+metadata(description="BOSCH BMI270 IMU driver.", version="1.0.1")
```

Comment:
For code that just has a change to the comments, I don't think we should update the version number.  It uses up a small amount of storage to store each version, and the compiled .mpy files anyway are not changed.

Reply:
reverted.